src/app/shared/components/event-detail/event-detail.component.html

```diff
@@ -0,0 +1,118 @@
+          <div class="item-content" fxFlex="1 0 78%">
+            <p>{{ event.id }}</p>
+          </div>
+          <div fxFlex="1 0 2%"></div>
+        </div>
+        <div class="detail-item" fxLayout="row" fxLayout.xs="column" fxLayoutAlign="space-between start"
+          fxLayoutGap="15px">
+          <span class="item-title" fxFlex="1 0 20%">时间</span>
+          <div class="item-content" fxFlex="1 0 78%">
+            <p>{{ event.time | date:'y年M月d日 H:mm' }}</p>
+          </div>
+          <div fxFlex="1 0 2%"></div>
+        </div>
+        <div class="detail-item" fxLayout="row" fxLayout.xs="column" fxLayoutAlign="space-between start"
+          fxLayoutGap="15px">
+          <span class="item-title" fxFlex="1 0 20%">地点</span>
+          <div class="item-content" fxFlex="1 0 78%">
+            <p>{{ event.location }}</p>
+          </div>
+          <div fxFlex="1 0 2%"></div>
+        </div>
+        <div class="detail-item" fxLayout="row" fxLayout.xs="column" fxLayoutAlign="space-between start"
+          fxLayoutGap="15px">
+          <span class="item-title" fxFlex="1 0 20%">时长</span>
+          <div class="item-content" fxFlex="1 0 78%">
+            <p>{{ event.num_hours | number:"0.1-1"  }}</p>
+          </div>
+          <div fxFlex="1 0 2%"></div>
+        </div>
+        <div class="detail-item" fxLayout="row" fxLayout.xs="column" fxLayoutAlign="space-between start"
+          fxLayoutGap="15px">
+          <span class="item-title" fxFlex="1 0 20%">参与人数</span>
+          <div class="item-content" fxFlex="1 0 78%">
+            <p>{{ event.num_participants }}</p>
+          </div>
+          <div fxFlex="1 0 2%"></div>
+        </div>
+        <div class="detail-item" fxLayout="row" fxLayout.xs="column" fxLayoutAlign="space-between start"
+          fxLayoutGap="15px">
+          <span class="item-title" fxFlex="1 0 20%">截止时间</span>
+          <div class="item-content" fxFlex="1 0 78%">
+            <p>{{ event.deadline | date:'y年M月d日 H:mm' }}</p>
+          </div>
+          <div fxFlex="1 0 2%"></div>
+        </div>
+        <div class="detail-item" fxLayout="row" fxLayout.xs="column" fxLayoutAlign="space-between start"
+          fxLayoutGap="15px">
+          <span class="item-title" fxFlex="1 0 20%">登记数</span>
+          <div class="item-content" fxFlex="1 0 78%">
+            <p>{{ event.num_enrolled }}</p>
+          </div>
+          <div fxFlex="1 0 2%"></div>
+        </div>
+        <div class="detail-item" fxLayout="row" fxLayout.xs="column" fxLayoutAlign="space-between start"
+           fxLayoutGap="15px">
+           <span class="item-title" fxFlex="1 0 20%">描述</span>
+           <div class="item-content" fxFlex="1 0 78%" [innerHTML]="description">
+           </div>
+           <div fxFlex="1 0 2%"></div>
+       </div>
+        <div fxLayout="row" fxLayoutAlign="end stretch" fxLayoutGap="5px">
+          <ng-container *ngIf="event.enrollments_status && eventDetailType === checkEventDetailType.USER">
```

Comment:
`enrollments_status` is bad, we can't tell whether user has enrolled, change it to `enrolled`, BTW, `overdue_status` is also bad, change it to `expired`, you can update these names in another PR.